src/HTSPTypes.h

```diff
@@ -54,6 +54,26 @@ typedef enum {
   DVR_AUTOREC_RECORD_ONCE_PER_DAY = 5
 } dvr_autorec_dedup_t;
 
+typedef enum {
+  DVR_RET_DVRCONFIG = 0,
+  DVR_RET_1DAY      = 1,
+  DVR_RET_3DAY      = 3,
+  DVR_RET_5DAY      = 5,
+  DVR_RET_1WEEK     = 7,
+  DVR_RET_2WEEK     = 14,
+  DVR_RET_3WEEK     = 21,
+  DVR_RET_1MONTH    = 31,
+  DVR_RET_2MONTH    = 62,
+  DVR_RET_3MONTH    = 92,
+  DVR_RET_6MONTH    = 183,
+  DVR_RET_1YEAR     = 366,
+  DVR_RET_2YEARS    = 731,
+  DVR_RET_3YEARS    = 1096,
+  DVR_RET_ONREMOVE  = INT32_MAX-1, // for retention only
+  DVR_RET_SPACE     = INT32_MAX-1, // for removal only
+  DVR_RET_FOREVER   = INT32_MAX
```

Comment:
I think you should put meaningful comments behind the first and also the last three values so that it becomes clear what these values do actually mean.